README.rst

```diff
@@ -18,3 +18,10 @@ Prerequisites
 * `Traits <https://pypi.python.org/pypi/traits>`_
 * `NumPy <https://pypi.python.org/pypi/numpy>`_
 * `SciPy <https://pypi.python.org/pypi/scipy>`_
+
+Development Environment Setup
+-------------
+1. Setup the EDM (with click, setuptools and coverage): edm install -e bootstrap click, setuptools, coverage
```

Comment:
Hi Mark, which command wasn't working? I just tested the procedure (running the instructions consecutively), the output seems fine (On MacOS Monterey 12.6.3):
<img width="733" alt="Screen Shot 2023-03-14 at 2 42 37 PM" src="https://user-images.githubusercontent.com/102019577/224907139-5aa3ac00-3839-486f-99b3-495cbe6b4d4b.png">

Reply:
Apologies. It's not normal to put commas in the EDM install list, and I mistakenly assumed that this wouldn't work at the EDM prompt, but apparently it does. That is, the normal form is:
```
edm install -e name_of_your_scimath_env click setuptools coverage
```
rather than
```
edm install -e name_of_your_scimath_env click, setuptools, coverage
```